models/intel/person-vehicle-bike-detection-2000/description/person-vehicle-bike-detection-2000.md

```diff
@@ -0,0 +1,52 @@
+# person-vehicle-bike-detection-2000
+
+## Use Case and High-Level Description
+
+This is a person, vehicle, bike detector that is based on MobileNetV2
+backbone with two SSD heads from 1/16 and 1/8 scale feature maps and clustered
+prior boxes for 256x256 resolution.
+
+## Example
+
+![](./person-vehicle-bike-detection-2000.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| AP @ [ IoU=0.50:0.95 ]          | 0.165 (internal test set)                 |
+| GFlops                          | 0.787                                     |
+| MParams                         | 1.821                                     |
+| Source framework                | PyTorch\*                                 |
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Performance
+
+## Inputs
+
+Name: `input`, shape: [1x3x256x256] - An input image in the format [BxCxHxW],
+where:
+
+- B - batch size
+- C - number of channels
+- H - image height
+- W - image width
+
+Expected color order is BGR.
+
+## Outputs
+
+The net outputs blob with shape: [1, 1, N, 7], where N is the number of detected
+bounding boxes. Each detection has the format
+  [`image_id`, `label`, `conf`, `x_min`, `y_min`, `x_max`, `y_max`], where:
+  - `image_id` - ID of the image in the batch
+  - `label` - predicted class ID
```

Comment:
Where can I find label map? 
Since there are few classes, best way is to add mapping in documentation.

Reply:
added